modules/rosa-hcp-aws-privatelink-create-cluster.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-aws-privatelink-creating-cluster.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-aws-privatelink-create-cluster_{context}"]
+= Creating an AWS PrivateLink cluster
+
+You can create a private cluster on {hcp-title} using the {product-title} (ROSA) CLI, `rosa`.
+
+[NOTE]
+====
+AWS PrivateLink is supported on existing VPCs only.
+====
+
+.Prerequisites
+
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You have installed and configured the latest {product-title} (ROSA) CLI, `rosa`, on your installation host.
+* You have created a VPC with at least one private subnet.
+
+.Procedure
+
+Creating a cluster with {hcp} can take around 10 minutes.
+
+. You can create a {hcp-title} cluster with multiple availability zones (Multi-AZ). Your machine's classless inter-domain routing (CIDR) must match your virtual private cloud's CIDR. See https://docs.openshift.com/container-platform/4.14/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-requirements_installing-aws-vpc[Requirements for using your own VPC] and link:https://docs.openshift.com/container-platform/4.14/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-validation_installing-aws-vpc[VPC Validation] for more information.
++
+[IMPORTANT]
+====
+If you use a firewall, you must configure it so that {product-title} can access the sites that it requires to function.
+
+For more information, see the AWS PrivateLink firewall prerequisites section.
+====
+
+. Create the account-wide IAM roles by running the following command:
++
+[source,terminal]
+----
+$ rosa create account-roles --hosted-cp
+----
+
+. Create the OIDC configuration by running the following command:
++
+[source,terminal]
+----
+$ rosa create oidc-config --mode=auto --yes
+----
++
+Save the OIDC configuration ID since you need it to create the operator roles.
```

Comment:
Per the [ISG word usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s) entry on "since", it's best to only use "since" to refer to time in order to help translation:
```suggestion
Save the OIDC configuration ID because you need it to create the operator roles.
```